### DISCUSSION TITLE
Hide infrequently used IATI Standard sub-elements to simplify data entry

### DISCUSSION THREAD

**Discussion by emmajclegg** [younginnovations/iatipublisher#1444] [General]
IATI Publisher currently shows (almost) all available sub-elements and attributes when populating elements from the activity overview page.

This is particularly overwhelming for the `transactions` and `results` elements - i.e. there are 36 data entry boxes to add a transaction [here](https://iatipublisher-staging.yipl.com.np/activity/940/transaction/create), and 18 for a result [here](https://iatipublisher-staging.yipl.com.np/activity/940/result/create).

What options are there to hide infrequently used sub-elements so that users don't see these? (but can find them if they do want to populate them) 

For example, in `activity-date,` the narrative sub-element is rarely needed and could be hidden behind an expandable menu.

![image](https://github.com/younginnovations/iatipublisher/assets/137316374/574fda52-6a85-41d1-9b3a-40c858c3bf3c)

It would be good to discuss whether this needs tackling element by element, or an approach could be agreed across IATI Publisher as a whole.

**Comment by praweshsth** [comment]
@emmajclegg 

For this, the first task will be to identify which field or group of fields that needs to be hidden and expanded if user wants to add them. How can we go ahead on this? Who will identify the fields in all the elements?

**Comment by emmajclegg** [comment]
Hi @praweshsth 

Some context may be helpful here, and we can keep this as a discussion for now. The IATI team want to work with IATI publishers and data users this year to define "IATI profiles", aiming to simplify publishing for smaller organisations. These "profiles" could be defined by type of organisation (e.g. donor, multilateral, non-governmental organisation, etc.) and each would have a recommended list of IATI fields to publish in their activity and organisation data.

This would mean that instead of all organisations being asked to publish the entire IATI Standard, we would recommend that small organisations, for example, publish a much smaller subset of data.

Once these "profiles" have been agreed, we could tailor IATI Publisher to look slightly different according to what profile the user selects (i.e. customising the user journey better). So, the motivation behind this issue of "hiding infrequently used fields" is a way to start thinking about how we highlight important fields and hide the unnecessary ones.

We're only starting to consult the IATI community now, so are not ready to define what all the profiles and "important fields" are exactly. But it would be useful to get any ideas from your team about how this could be implemented in future.
